sql_scripts/create_tabls.sql

```diff
@@ -0,0 +1,77 @@
+-- This table is about the details of authors of books in the library
+
+CREATE TABLE author 
+  ( 
+     authorid    INTEGER(255) COMMENT "id of the author",
+     afirstname  VARCHAR(20) COMMENT "first name of the author",
+     amiddlename VARCHAR(20) COMMENT "middle name of the author",
+     alastname   VARCHAR(20) COMMENT "last name of the author",
+     contact     INTEGER(255) COMMENT "phone number of the author",
+     PRIMARY KEY(authorid)    COMMENT "primary key of the table is author id"
+  ); 
+
+-- This table gives list of all the books in library
+  
+CREATE TABLE books 
+  ( 
+     id         INTEGER(255) COMMENT "id of the book",
+     title      VARCHAR(20) COMMENT "title of the book",
+     copies     INTEGER(255) COMMENT "number of copies of each book", 
+     bookstaken INTEGER(255) COMMENT "number of copies of each book taken by the students",
+     PRIMARY KEY(id)         COMMENT "primary key of the table is book id"
+  ); 
+-- This table gives the details of the students in the college
+
+CREATE TABLE student 
```

Comment:
Empty space line should be after table completion. Table should follow immediately after comment.